ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -42,6 +42,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       ENABLE_FORCE: "true"
       GC: "false"
+      ITERATION_MULTIPLIER_ENV: "6"
```

Comment:
@liqcui So, we want to reduce it from default 9 to 6? Any specific reason? Was the load too high with 9, or choosing lower load just to be safe?

Reply:
Yes, 9 is too high, the job will fail several times when run cluster-density-v2 job using 9.